src/jimgw/single_event/likelihood.py

```
@@ -11,13 +11,14 @@
 
 from jimgw.base import LikelihoodBase
 from jimgw.prior import Prior
-from jimgw.single_event.detector import Detector
+from jimgw.single_event.detector import Detector, GroundBased2G
```

Ruff (0.8.2)

`jimgw.single_event.detector.GroundBased2G` imported but unused

Remove unused import: `jimgw.single_event.detector.GroundBased2G`

(F401)